eva_sub_cli/__init__.py

```diff
@@ -15,3 +15,13 @@
 SUBMISSION_WS_VAR = 'SUBMISSION_WS_URL'
 ENA_WEBIN_ACCOUNT_VAR = 'ENA_WEBIN_ACCOUNT'
 ENA_WEBIN_PASSWORD_VAR = 'ENA_WEBIN_PASSWORD'
+
+
+def is_submission_dir_writable(submission_dir):
```

Comment:
I'm not a huge fan of putting actual functions/logic in `__init__.py`, can it maybe go in `main.py`?

Reply:
Utils is inevitable, embrace the utils :lotus_position: 

(I'm just kidding, it's up to you of course)

Reply:
Added now :)